ui/src/components/shared/CredentialRevokeModal.tsx

```diff
@@ -3,7 +3,8 @@ import { useState } from "react";
 
```

Comment:
Can you please update the literal:

> Revoking of a credential must be accompanied by publishing of issuer state in order for  the action to be recorded

as:

> Revoking of a credential must be accompanied by publishing of issuer state in order for  the action to be effective

To be consistent with the delete connection msg?